### DISCUSSION TITLE
Does edb have something like gdb-multiarch?

### DISCUSSION THREAD

**Discussion by colin-i** [eteran/edb-debugger#866] [General]
If I want to debug on a RPI 4 amd64 OS an x86 program I start with qemu-i386 -g port, connect with gdb-multiarch and target remote :port.
Is a similar approach with armhf edb?

**Comment by eteran** [comment]
If I've understood correctly, unfortunately, no, edb doesn't have anything quite equivalent to gdb server, so remote debugging isn't (yet) supported.

In the past we've discussed breaking out the debug core into a separate process, which would be the first step in supporting that... But no work has been done on it yet.

**Comment by colin-i** [comment]
Yes, on an arm64 OS Qemu is doing the x86 to arm translation and sends x86 informations to gdb-multiarch in what I said.  Qemu has a gdbstub(gdbserver) and gdb-multiarch sends/gets informations from Qemu.

**Reply by colin-i** [reply]
The solution for me was to use qemu and gdb multiarch with gef extension (https://github.com/hugsy/gef).
